src/main/java/racingcar/service/validator/InputValidator.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.service.validator;
+
+import java.util.List;
+
+public class InputValidator {
+    private static final int MAX_CAR_NAME = 5;
+    private static final int MIN_CAR_NAME = 1;
+    private static final int MIN_ATTEMPTS = 1;
+
+    public void isStringNull(String input) {
+        if (input.isEmpty()) {
+            throw new IllegalArgumentException("입력값이 없습니다.");
+        }
+    }
+
+    public void validateCarNames(List<String> carNames) {
+        carNames.stream().filter(name -> name.length() > MAX_CAR_NAME).forEach(name -> {
+            throw new IllegalArgumentException("자동차 이름은 5자 이하여야 합니다.");
+        });
```

Comment:
자릿수 제한을 상수 처리하였으면 예외 메시지 속 숫자도 상수를 활용하면 좋을 것 같습니다! 

```suggestion
    public void validateCarNames(List<String> carNames) {
        carNames.stream().filter(name -> name.length() > MAX_CAR_NAME).forEach(name -> {
            throw new IllegalArgumentException("자동차 이름은 " + MAX_CAR_NAME + "자 이하여야 합니다.");
        });
```

Reply:
오 이 방법은 생각도 못해봤어요

배워갑니다.

Reply:
문자열의 + 연산 대신 `String.format`을 활용하면 더 깔끔하게 작성할 수도 있을 것 같아요!

Reply:
오 그 방법도 배워갑니다!!